backend/easyrtc-server/server.js

```diff
@@ -18,6 +18,8 @@ var webServer = http.createServer(app);
 // Start Socket.io so it attaches itself to Express server
 var socketServer = socketIo.listen(webServer, {"log level":1});
 
+easyrtc.setOption("demosEnable", "false");
+easyrtc.setOption("logLevel", "debug");

```

Comment:
Duplicate line?

Reply:
Both lines change a different option. The first is somewhat useless know and could be removed, but they are not duplicate:
https://easyrtc.com/docs/easyrtc_server_configuration.php

Reply:
Both lines change a different option. The first is somewhat useless now and could be removed, but they are not duplicate:
https://easyrtc.com/docs/easyrtc_server_configuration.php